app/models/category.rb

```diff
@@ -15,7 +15,7 @@ class Category < ApplicationRecord
 
   validates :name, presence: true
 
-  scope :nonempty, -> {joins(:runs)}
+  scope :nonempty, -> {joins(:game).group('categories.id').joins(:runs)}
```

Comment:
Would a `joins(:runs).distinct` suffice? Just a bit simpler and more efficient if so.

Reply:
`:runs`, not `:game` :P Think we lost our original goal